### ISSUE TITLE
EditableDialog not showing labels of checkboxes

### ISSUE THREAD

**Issue by roland4432** [pimcore/pimcore#7413]
# Bug Report

### Expected behavior
When using checkboxes in editable dialogs, their labels won't be shown (still be added to admin translation tho)
### Actual behavior

The label is only rendered on the first call and empty from the 2nd call on

### Steps to reproduce

When loading the page for the first time, the label will be shown. After a reload, neither the original label nor the translated value will be shown.
Create brick, use the new interface / function to implement the editable dialog. Add a checkbox to the dialog just like stated in the documentation.

### Code example
```php
public function getEditableDialogBoxConfiguration(Document\Editable $area, ?\Pimcore\Model\Document\Editable\Area\Info $info): EditableDialogBoxConfiguration {
        $config = new EditableDialogBoxConfiguration();
        $config
            ->setWidth(700)
            ->setItems([
               [
                     'type' => 'checkbox',
                     'label' => 'my checkboxlabel',
                     'name' => 'mycheckbox',
               ],
            ])
        ;

        return $config;
    }
```

```javascript
    //AdminBundle/Resources/public/js/pimcore/document/editables/checkbox.js
    ...
    renderInDialogBox: function (config) { //<- config is undefined since no value is passed to this call (not used anyways)
        if(this.config['dialogBoxConfig'] &&  // this.config is empty when this function is being called, so no label is shown
            (this.config['dialogBoxConfig']['label'] || this.config['dialogBoxConfig']['name'])) {
            this.config["label"] = this.config['dialogBoxConfig']['label'] || this.config['dialogBoxConfig']['name'];
        }

        this.render();
    },
    ...
```

**Comment by NiklasBr** [comment]
> The label is only rendered on the first call and empty from the 2nd call on

Happens in 6.8.4 for me as well.

**Comment by pargy** [comment]
This way it seems to work for me:

```php

[
  'type' => 'checkbox',
  'name' => 'foo',
  'config' => [
    'label' => 'bar'
  ]
]
```

**Comment by brusch** [comment]
Unfortunately I'm not able to reproduce the issue ... see: 

![05-11-_2020_10-34-42](https://user-images.githubusercontent.com/142037/98223584-9dca6b00-1f52-11eb-96c1-6863710ec380.gif)


Used this config: 
https://pimcore.com/docs/pimcore/current/Development_Documentation/Documents/Editables/Areablock/Bricks.html#page_Simple-Example-Config

**Comment by roland4432** [comment]
Well.. the issue just went *poof* .. gone :D
I couldn't dig into the cause of the problem but I can approve, that the behavior is now correct and all the checkbox labels are shown as intended.

@NiklasBr can you confirm that the labels are shown now?

**Comment by NiklasBr** [comment]
Still happening here with Reload On Close active

**Comment by brusch** [comment]
Just tried with
```php
<?php

namespace AppBundle\Document\Areabrick;

use Pimcore\Extension\Document\Areabrick\EditableDialogBoxConfiguration;
use Pimcore\Extension\Document\Areabrick\EditableDialogBoxInterface;
use Pimcore\Model\Document;
use Pimcore\Model\Document\Editable\Area\Info;

class WysiwygWithImages extends AbstractAreabrick implements EditableDialogBoxInterface
{
    /**
     * {@inheritdoc}
     */
    public function getName()
    {
        return 'WYSIWYG w. Images';
    }

    public function getEditableDialogBoxConfiguration(Document\Editable $area, ?Info $info): EditableDialogBoxConfiguration
    {
        $config = new EditableDialogBoxConfiguration();
        $config->setWidth(600);
        $config->setReloadOnClose(true);
        $config->setItems([
            [
                'type' => 'checkbox',
                'name' => 'myDialogCheckbox',
                'label' => 'This is the checkbox label',
            ]
        ]);


        return $config;
    }
}
```

works like a charm on the demo ...

**Comment by NiklasBr** [comment]
```php
<?php

declare(strict_types=1);

namespace AppBundle\Document\Areabrick;

use Pimcore\Extension\Document\Areabrick\EditableDialogBoxConfiguration;
use Pimcore\Extension\Document\Areabrick\EditableDialogBoxInterface;
use Pimcore\Model\Document\Editable;
use Pimcore\Model\Document\Editable\Area\Info;

class Fiftyfiftybrick extends AbstractBaseBrick implements EditableDialogBoxInterface
{
    public function getName(): string
    {
        return 'brick.name.fifty-fifty';
    }

    /**
     * {@inheritdoc}
     */
    public function getDescription()
    {
        return $this->translate('brick.description.fifty-fifty');
    }

    public function getIcon(): string
    {
        return '/bundles/pimcoreadmin/img/flat-color-icons/book.svg';
    }

    public function getEditableDialogBoxConfiguration(Editable $area, ?Info $info): EditableDialogBoxConfiguration
    {
        $config = new EditableDialogBoxConfiguration();
        $config->setWidth(600);
        $config->setReloadOnClose(true);
        $config->setItems([
            [
                'type' => 'checkbox',
                'label' => 'brick.options.label.inverse-position',
                'name' => 'imagePosition',
            ],
        ]);

        return $config;
    }
}
```

100% possible to reproduce in above code

**Comment by brusch** [comment]
Fixed by #7471
